gcsio/pom.xml

```diff
@@ -142,27 +160,38 @@
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigdataoss</groupId>
+      <artifactId>util-hadoop</artifactId>
+      <version>${hadoop.identifier}-2.2.3-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
 
     <!-- Test dependencies -->
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+      <scope>test</scope>
```

Comment:
It's already defined in parent pom, please remove here and below.

Reply:
will do. Thought to make it clear after the `<!-- Test dependencies -->` comment :)